src/components/AvatarCropModal/AvatarCropModal.tsx

```diff
@@ -239,65 +237,69 @@ function AvatarCropModal({imageUri = '', imageName = '', imageType = '', onClose
         onBegin: () => {
             'worklet';
 
-            isPressableEnabled.value = false;
+            isPressableEnabled.set(false);
         },
         onChange: (event: GestureUpdateEvent<PanGestureHandlerEventPayload & PanGestureChangeEventPayload>) => {
             'worklet';
 
-            const newSliderValue = clamp(translateSlider.value + event.changeX, [0, sliderContainerSize]);
+            const newSliderValue = clamp(translateSlider.get() + event.changeX, [0, sliderContainerSize]);
             const newScale = newScaleValue(newSliderValue, sliderContainerSize);
 
-            const differential = newScale / scale.value;
+            const differential = newScale / scale.get();
 
-            scale.value = newScale;
-            translateSlider.value = newSliderValue;
+            scale.set(newScale);
+            translateSlider.set(newSliderValue);
 
-            const newX = translateX.value * differential;
-            const newY = translateY.value * differential;
+            const newX = translateX.get() * differential;
+            const newY = translateY.get() * differential;
             updateImageOffset(newX, newY);
         },
         onFinalize: () => {
             'worklet';
 
-            isPressableEnabled.value = true;
+            isPressableEnabled.set(true);
         },
     };
 
     // This effect is needed to prevent the incorrect position of
     // the slider's knob when the window's layout changes
     useEffect(() => {
-        translateSlider.value = interpolate(scale.value, [CONST.AVATAR_CROP_MODAL.MIN_SCALE, CONST.AVATAR_CROP_MODAL.MAX_SCALE], [0, sliderContainerSize]);
-    }, [scale.value, sliderContainerSize, translateSlider]);
+        translateSlider.set(interpolate(scale.get(), [CONST.AVATAR_CROP_MODAL.MIN_SCALE, CONST.AVATAR_CROP_MODAL.MAX_SCALE], [0, sliderContainerSize]));
+    }, [scale, sliderContainerSize, translateSlider]);
 
     // Rotates the image by changing the rotation value by 90 degrees
     // and updating the position so the image remains in the same place after rotation
     const rotateImage = useCallback(() => {
-        rotation.value -= 90;
+        runOnUI(() => {
+            rotation.set((value) => value - 90);
 
-        // Rotating 2d coordinates by applying the formula (x, y) → (-y, x).
-        [translateX.value, translateY.value] = [translateY.value, translateX.value * -1];
+            const oldTranslateX = translateX.get();
```

Comment:
`const [oldTranslateX, oldTranslateY] = [translateX.get(), translateY.get()];
translateX.set(oldTranslateY);
translateY.set(oldTranslateX * -1);`

What do you think about writing it this way? seems more readable to me